packages/admin/admin-theme/src/componentsTheme/MuiMenuItem.tsx

```diff
@@ -0,0 +1,13 @@
+import { mergeOverrideStyles } from "../utils/mergeOverrideStyles";
+import { GetMuiComponentTheme } from "./getComponentsTheme";
+
+export const getMuiMenuItem: GetMuiComponentTheme<"MuiMenuItem"> = (component, theme) => ({
+    ...component,
+    styleOverrides: mergeOverrideStyles<"MuiMenuItem">(component?.styleOverrides, {
+        root: {
+            "& .MuiListItemIcon-root": {
+                minWidth: 0,
+            },
```

Comment:
I think it would make sense to set this for every `ListItemIcon` (in `getListItemIcon.ts`), otherwise those inside `MenuItem` and `ListItem` could differ in styling.